README.md

```diff
@@ -8,59 +8,59 @@ lighting-rfc repository, specifically within
+ability of participants on the network from easily censoring payments, as the
 participants are not aware of the final destination of any given payment.
-Additionally, by encoding payment routes within a mix-net like packet, we are
-able to achieve the following security and privacy features: 
+Additionally, by encoding payment routes within a mix-net type of packet, we
+are able to achieve the following security and privacy features: 
 
-  * Participants in a route don't know their exact position within the route
-  * Participants within a route don't know the source of the payment, nor the
+  * participants in a route don't know their exact position within the route
+  * participants within a route don't know the source of the payment, nor the
     ultimate destination of the payment
-  * Participants within a route aren't away _exactly_ how many other
+  * participants within a route aren't aware _exactly_ how many other
     participants were involved in the payment route
-  * Each new payment route is computationally indistinguishable from any other
+  * each new payment route is computationally indistinguishable from any other
     payment route
 
 Our current onion routing protocol utilizes a message format derived from
-[Sphinx](http://www.cypherpunks.ca/~iang/pubs/Sphinx_Oakland09.pdf). In order
+[Sphinx](http://www.cypherpunks.ca/~iang/pubs/Sphinx_Oakland09.pdf).  In order
 to cater Sphinx's mix-format to our specification application, we've made the
 following modifications: 
 
-  * We've added a MAC over the entire mix-header as we have no use for SURB's
-    (single-use-reply-blocks) in our protocol.
-  * Additionally, the end-to-end payload to the destination has been removed in
+  * added a MAC over the entire mix-header as we have no use for SURBs
+    (single-use-reply-blocks) in our protocol
+  * additionally, the end-to-end payload to the destination has been removed in
     order to cut down on the packet-size, and also as we don't currently have a
-    use for a large message from payment sender to recipient.
-  * We've dropped usage of LIONESS (as we don't need SURB's), and instead
-    utilize chacha20 uniformly throughout as a stream cipher.
-  * Finally, the mix-header has been extended with a per-hop-payload which
-    provides each hops with exact instructions as to how and where to forward
-    the payment. This includes the amount to forward, the destination chain,
-    and the time-lock value to attach to the outgoing HTLC.
+    use for a large message from the payment sender to the recipient
+  * dropped usage of LIONESS (as we don't need SURBs), and instead utilize
+    chacha20 uniformly throughout as a stream cipher
+  * finally, the mix-header has been extended with a per-hop-payload which
+    provides each hop with exact instructions as to how and where to forward
+    the payment (this includes the amount to forward, the destination chain,
```

Comment:
**. (** 

did not change the semantics, change the signal is necessary even in this case? (or this?)